requirements.txt

```
@@ -29,4 +29,4 @@ schema==0.7.1
 simplegeneric==0.8.1
 traitlets==4.3.1
 wcwidth==0.1.7
-wsgiref==0.1.2
+wsgiref==0.1.2; python_version < '3.0'
```

Nice!  Thanks for doing this.
```suggestion
wsgiref==0.1.2; python_version < '3.2'
```
https://pypi.org/project/wsgiref/